lib/tasks/tenants.rake

```diff
@@ -1,55 +1,72 @@
 # frozen_string_literal: true
 
+# Temporarily disable Sentry for this task
+Raven.configure do |config|
+  config.silence_ready = true
+end
```

Comment:
Will this disable Raven for the running application while the task is running?

Reply:
Good point, I believe so.